client/src/components/hero/HeroContentDesktop.tsx

```diff
@@ -39,6 +41,10 @@ const UnstyledHeroContentDesktop: FunctionComponent<HomeDesktopProps> = ({
     return 15;
   }, [isMedium]);
 
+  const announcementID = (id: string) => {
```

Comment:
rename the function that represents what this does.
This sounds like variable.

Reply:
Fixed!